docs/events/AnalysisRunStateChange/examples/ARSC__example2.json

```diff
@@ -8,37 +8,16 @@
   "region": "ap-southeast-2",
   "resources": [],
   "detail": {
+    "id": "hash(values) e.g. 97534601940f17ebcfee02e9df998244",
+    "version": "<semver> e.g. 1.0.0",
+    "timestamp": "2025-05-01T09:25:44Z",
     "orcabusId": "anr.76J5N2J83RED7387G9374NGDBB",
     "analysisRunName": "<unique analysis name>",
-    "timestamp": "2025-05-01T09:25:44Z",
     "status": "DRAFT",
     "analysis": {
       "orcabusId": "ana.76J5N2J83RED7387G9374NGDBA",
       "name": "<analysis name>, e.g. oncoanalyser-dna",
       "version": "<analysis version>, e.g. 1.0.4"
-    },
-    "libraries": [
```

Comment:
I think libraries are mandatory even in a minimal example. 
Not sure there's a valid case where you'd want to initiate an AnalysisRun if you don't even have the libraries yet?

Reply:
Let go stricter. Relaxing is easy. I will update [ARSC libraries](https://github.com/OrcaBus/service-workflow-manager/blob/bd6b179545ce4bf15bf5a352a689496c6faae2b2/docs/events/AnalysisRunStateChange/AnalysisRunStateChange.schema.yaml#L36-L43) mandatory property.